README.md

```diff
@@ -396,6 +396,12 @@ Loans include information from the item, including locations, holdingsRecordId a
 
 Requests include information from the item, including holdingsRecordId and instanceId.
 
+## Known limitations
```

Comment:
There is already a [known limitations](https://github.com/folio-org/mod-circulation/blob/master/README.md#known-limitations) section in the readme, please can this be combined together with that

Reply:
Sure, I'll move it here.